actions/middleware.go

```diff
@@ -102,6 +113,22 @@ func CheckAuthentication(appConfig *config.AppConfig, bux bux.ClientInterface, r
 	return req, dictionary.ErrorMessage{}
 }
 
+func VerifyCallbackToken(appConfig *config.AppConfig, req *http.Request) (*http.Request, dictionary.ErrorMessage) {
+	const BEARER_SCHEMA = "Bearer "
+	authHeader := req.Header.Get("Authorization")
+	if authHeader == "" {
+		return req, dictionary.GetError(dictionary.ErrorAuthenticationCallback, "missing auth header")
+	}
+
+	providedToken := authHeader[len(BEARER_SCHEMA):]
```

Comment:
Potential Index Out of Range Error: If `authHeader` is shorter than "Bearer ", the slicing operation `authHeader[len(BEARER_SCHEMA):]` could cause an index out of range error.

Reply:
Another, maybe reasonable (or not) issue is that: 
- What if somebody put different string at the beginning of the authorization header with the same length as "Bearer"?This code will still accept this header, even if it's another auth-scheme.

Reply:
fixed